ra2ce/analysis/analysis_config_data/analysis_config_data.py

```diff
@@ -93,7 +95,10 @@ class AnalysisSectionLosses(AnalysisSectionBase):
     # losses
     traffic_cols: list[str] = field(default_factory=list)
     production_loss_per_capita_per_hour: float = math.nan
-    part_of_day: PartOfDayEnum = field(default_factory=lambda: PartOfDayEnum.DAY)
+    traffic_period: TrafficPeriodEnum = field(
+        default_factory=lambda: TrafficPeriodEnum.DAY
+    )
+    hours_per_traffic_period: int = 0
```

Comment:
```suggestion
    hours_per_traffic_period: int = 24
```

Reply:
since the traffic_period has a default value of TrafficPeriodEnum.DAY, then hours_per_... should also have the default value of 24

Reply:
Doing this would make it harder to see if a warning is needed in the losses analysis. I'll assume 24 there, logging a warning if the value is not set (still 0) and the traffic period is not DAY.

Reply:
ok, but if the traffic period is Day (the default), then the hours_per_traffic_period remains 0, right?

Reply:
Agree. Made a small adjustment in `losses_base` for this